torch/csrc/distributed/rpc/script_call.cpp

```diff
@@ -13,10 +13,35 @@ ScriptCall::ScriptCall(
     std::vector<at::IValue>&& args)
     : op_(std::move(op)), stack_(args) {}
 
+ScriptCall::ScriptCall(
+    const c10::QualifiedName& qualifiedName,
+    std::vector<at::IValue>&& args)
+    : qualifiedName_(qualifiedName), stack_(args) {}
+
+bool ScriptCall::hasOp() const {
+  if (op_) {
```

Comment:
nit: just do `return op_`? Same for `hasQualifiedName` below.

Reply:
will clean it up